src/main/java/com/amihaiemil/docker/UserAgentRequestHeader.java

```diff
@@ -34,17 +37,18 @@
  * User Agent Request Header Interceptor.
  * @author Boris Kuzmic (boris.kuzmic@gmail.com)
  * @since 0.0.7
- * @todo #244:30min Replace hardcoded version with the one loaded from
- *  properties file. For example, create new version.properties file
- *  with property: build.version=${project.version}, add to src/main/resources
- *  and Maven filtering will do the rest.
  */
 final class UserAgentRequestHeader extends RequestDefaultHeaders {
 
     /**
-     * Version constant.
+     * Version properties file.
      */
-    private static final String VERSION = "0.0.8-SNAPSHOT";
+    private static final String VERSION_FILE = "version.properties";
```

Comment:
@bkuzmic I would just like to rename this file to ``config.properties``, not ``version.properties``, since in the future it may hold more stuff, right?

Reply:
@amihaiemil Makes sense.